### PR TITLE
updating workflow

### PR DESCRIPTION
**Pull request by fiddlermikey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+This repository contains a Universal Orchestrator Capability which is a plugin to the Keyfactor Universal Orchestrator. Within the Keyfactor Platform, Orchestrators are used to manage “certificate stores” &mdash; collections of certificates and roots of trust that are found within and used by various applications.
+
+The Universal Orchestrator is part of the Keyfactor software distribution and is available via the Keyfactor customer portal. For general instructions on installing Capabilities, see the “Keyfactor Command Orchestrator Installation and Configuration Guide” section of the Keyfactor documentation. For configuration details of this specific Capability, see below in this readme.
+
+The Universal Orchestrator is the successor to the Windows Orchestrator. This Capability plugin only works with the Universal Orchestrator and does not work with the Windows Orchestrator.
+
+---
+
 # Remote JKS
 ## orchestrator
```
Comment (doebrowsk) [inline README.md:18]: This generated README doesn't look right. We shouldn't have the old tempating content showing up below the line `---`.

**Review by doebrowsk — CHANGES_REQUESTED** [review]

File: .github/workflows/keyfactor-starter-workflow.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  call-create-github-release-workflow:
+    uses: Keyfactor/actions/.github/workflows/github-release.yml@main
+
+  call-dotnet-build-and-release-workflow:
+    needs: [call-create-github-release-workflow]
+    uses: Keyfactor/actions/.github/workflows/dotnet-build-and-release.yml@main
+    with:
+      release_version: ${{ needs.call-create-github-release-workflow.outputs.release_version }}
+      release_url: ${{ needs.call-create-github-release-workflow.outputs.release_url }}
+      release_dir: jks-remote-orchestrator/bin/Release/BUILD_TARGET # TODO: set build output directory to upload as a release, relative to checkout workspace
```
[the remark this answers is not in the document]
Comment (fiddlermikey) [reply .github/workflows/keyfactor-starter-workflow.yml:14]: I do not see an output directory defined but PROJECT_FOLDER: 'JavaKeyStoreSSH' is defined. Should this release dir then be: jks-remote-orchestrator/bin/Release/JavaKeyStoreSSH

File: .github/workflows/keyfactor-extension-release.yml [hunk 3]
```diff
@@ -1,134 +0,0 @@
...
-      - name: Execute MSBuild Commands
-        run: |
-          MSBuild.exe $Env:SOLUTION_PATH -p:RestorePackagesConfig=false -p:Configuration=Release       
-      
-      - name: Archive Files
-        if: ${{ success() }}
-        run: |
-           md ${{ github.workspace }}\zip\Keyfactor
-           Compress-Archive -Path `
-           ${{ env.SOLUTION_FOLDER }}\${{ env.PROJECT_FOLDER }}\bin\Release\netcoreapp3.1\* `
```
Comment (doebrowsk) [inline .github/workflows/keyfactor-extension-release.yml:112]: This is the relevant line that would package the build output in the old workflow @fiddlermikey

File: .github/workflows/keyfactor-starter-workflow.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (doebrowsk) [reply .github/workflows/keyfactor-starter-workflow.yml]: No, our BUILD_TARGET should either be a dotnet type (netstandard2.0 or netcoreapp3.1) or nothing if output from a dotnet framework project.

File: .github/workflows/keyfactor-extension-release.yml (hunk 3 above)
Comment (fiddlermikey) [reply .github/workflows/keyfactor-extension-release.yml:112]: Updated the output folder

**Pull request merged**